src/libraries/System.Reflection.Emit/tests/DynamicMethodTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Reflection.Emit.Tests;
+
+public class DynamicMethodTests
```

Comment:
I was surprised not to find an existing test file for this; is there one in another place that I just missed?

Reply:
DynamicMethod / RTDynamicMethod split is a left-over from Code Access Security that was removed in .NET Core. We did not bring any Core Access Security tests over.

Reply:
Not sure why this didn't come up in my search, but thanks! Once there is some alignment about whether this approach is even ok, I'll relocate the test to that project.